README.md

```diff
@@ -226,6 +226,24 @@ Tips:
 1. For structured facts index into the specific fact value otherwise the entire array will come over as a string and ultimately be difficult to use.
 2. Dynamic facts such as CPU usage, Uptime, and others that are expected to change each run are not ideal for tagging.  Static facts that are expected to stay for the life of a node are best candidates for tagging.
 
+### Log collection
+
+Collecting logs is disabled by default, enable it in your Puppet configuration file by setting `logs_enabled: true`. Additional file paths can also be configured.
+
+```
+  class { 'datadog_agent::integrations::logs' :
+    logs => [
+      {
+        'type' => 'file',
+        'path' => '/var/log/afile.log',
+      },
+      {
+        'type' => 'docker',
+      },
+    ],
+ }
+```
```

Comment:
If the log collection is related to an existing integration, the recommendation we give in our docs [1] is that logs are configured in the same integration yaml file. This still works, though, but we might wanna align this with the linked docs and mention they can configure this in the integration itself.

[1] https://docs.datadoghq.com/agent/logs/?tab=tailfiles#custom-log-collection.

Reply:
hey @albertvaka these are the relevant docs, but this configuration is for Puppet specifically. where do you suggest this is added?

Reply:
It can be passed in the `logs` inside the `integrations`.
```
class { 'datadog_agent':
    api_key      => "<YOUR_DD_API_KEY>",
    integrations => {
        "ntp" => {
            init_config => {},
            instances => [{
                offset_threshold => 30,
            }],
            logs => [
                {
                    'type' => 'file',
                    'path' => '/var/log/afile.log',
                },
            ],
        },
    },
}
```

I agree it's a bit more cumbersome and I'm fine if we wanna keep it simple, but this way (configuring logs per-check) would be more in line with the general documentation.